src/lib/PPrint.hs

```diff
@@ -949,6 +958,7 @@ prettyPrecPrimCon con = case con of
   LabelCon name -> atPrec ArgPrec $ "##" <> p name
   ExplicitDict _ _ -> atPrec ArgPrec $ "ExplicitDict"
   DictHole _ e -> atPrec LowestPrec $ "synthesize" <+> pApp e
+  HandlerHole _ e -> atPrec LowestPrec $ "synthesize" <+> pApp e
```

Comment:
perhaps call it something else than `synthesize` to avoid confusion?  "resolve"?

Reply:
Handler holes removed in #1071